README.md

```diff
@@ -18,8 +18,8 @@ You can customize the banner with additional parameters by setting script attrib
     Default: ```'We stand with Palestine'```
 
 ```
-<script src="https://cdn.jsdelivr.net/gh/devkayes/we-stand-with-palestine@main/ribbon-banner.js" title="We support Palestine" async></script>
+<script src="https://cdn.jsdelivr.net/gh/samiulhafiz/we-stand-with-palestine@main/ribbon-banner.js" title="We support Palestine" async></script>
```

Comment:
Same here also! 

Re: While you merge your main branch into the source branch, you need to provide the source branch repository link, not your source branch link. Your code is completely fine, but when you request the PR, your `README.md` changes will overwrite mine. 

In that case, either you have to change the `README.md` content or make a change to the content in another branch and then submit the PR with the newly created branch again (best practice).  

The updated README.md content should be: `<script src="https://cdn.jsdelivr.net/gh/devkayes/we-stand-with-palestine@main/ribbon-banner.js" async></script>`

Let me know if you need help with that; thanks.